### PR TITLE
Migration to Node.js Org

### PR DESCRIPTION
**Pull request by UlisesGascon**
### Context

- Related to https://github.com/UlisesGascon/poc-nodejs-news-feeder/pull/8#issuecomment-1598209928 
- Close https://github.com/nodejs/admin/issues/803

### CODE REVIEW

**Comment by mhdawson** [comment]
> This will trigger the Github Pages deployment once this change is pushed to main

Not quite sure I follow. This is just a readme doc change should there be more ?

**Comment by UlisesGascon** [comment]
> Not quite sure I follow. This is just a readme doc change should there be more ?

@mhdawson I made a poor explanation. To deploy the repository content to Github Pages, it was necessary to add new commits to the `main` branch. This was done when I merged #9. Therefore, there is no need to worry anymore. 😉

File: README.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
 This process will generate a PR with the latest news, so we can change the content and decide when to merge it.
 
 ### Development
 
 ```bash
-git clone https://github.com/UlisesGascon/poc-nodejs-news-feeder
-cd poc-nodejs-news-feeder
+git clone https://github.com/nodejs/news-feeder
+cd news-feeder
```
Comment (mhdawson) [inline README.md:17]: plan to rename to match what we had in admin repo which was `nodejs-news-feeder`

Comment (UlisesGascon) [reply README.md]: solved in 72b97d9

File: README.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
-# poc-nodejs-news-feeder
+# Official Node.js News Feeder
 
 ### Usage
 
 This project is a proof of concept of a news feeder using NodeJS. It uses the Github API to fetch the relevant information from issues, releases..and then it generates a RSS feed with the latest news.
 
-The RSS feed is available at https://raw.githubusercontent.com/UlisesGascon/poc-nodejs-news-feeder/main/feed.xml
+The RSS feed is available at https://raw.githubusercontent.com/nodejs/news-feeder/main/feed.xml and https://ulisesgascon.github.io/news-feeder/feed.xml
```
[the remark this answers is not in the document]
Comment (UlisesGascon) [reply README.md]: yep, solved in d62d23fc6f9cc31e53184039fd0e3c2e883de0d9

**Pull request merged**